src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -60,6 +61,17 @@
             AutomationProperties.AutomationId="{Binding Id, Mode=OneWay, StringFormat='LicenseOperator_{0}'}"
             Text="{Binding Text}" />
       </DataTemplate>
+
+      <Style x:Key="TreeViewItemHeaderStyle" TargetType="TreeViewItem">
```

Comment:
Are you moving styles out to this dictionary just for readability? I didn't understand how it related to the new Brushes, so assuming it's just a refactoring.
It's probably negligible, but if anything, I thought it was better perf to have resources as close to the control as possible if they're not shared.

Reply:
That's a great question! I took the opportunity to clean up the duplication by factoring these styles into their own style resources for readability as you pointed out. Putting them up in the control's resource dictionary is just for better organization and following common practice.

I believe resource resolution happens from a larger to narrower scope (top-down) rather than bottom-up, but I do know for certain that perf is negligible in either case.

Reply:
generally in WPF: 
resource dictionary resolution starts at the element in the tree, and walks towards the root of the window/control.
then jumps to app resources, etc...